website/content/roadmap.md

```diff
@@ -3,63 +3,71 @@ title: "Roadmap"
+> unsatisfied with SQL. We currently have an initial working version for the
+> intrepid early user.
+>
+> I'm hoping we can build a beautiful language, an app that's approachable &
+> powerful, and a vibrant community. Many projects have reached the current stage
+> and fallen, so this requires compounding on what we've done so far.
+>
+> -- <cite>Maximilian Roos</cite>
 
 {{< columns >}}
 
-## Language design
+## Language
 
 Already since becoming public, the language has improved dramatically, thanks to
 the feedback of dozens of contributors. The current state of the basics is now
 stable and while we'll hit corner-cases, I expect we'll only make small changes
 to the existing features — even as we continue adding features.
 
-Feel free to post questions or continue discussions on [Language Design
-Issues](https://github.com/prql/prql/issues?q=is%3Aissue+is%3Aopen+label%3Alanguage-design).
+On this fundation we are planning to build advanced features like type checking,
+function currying, pivot/melt/wide_to_long/long_to_wide operations, operator overloading and
+[a few more](https://github.com/prql/prql/issues?q=is%3Aissue+is%3Aopen+label%3Alanguage-design).
+This will take time, because we want to build a consistent language that feels like it is
+made to last.
 
-## Documentation
+## Friendliness
 
-Currently the language documentation is at <https://prql-lang.org/book>.
+Currently the compiler is not sufficiently friendly, despite significant recent improvements.
+We'd like to make error messages better and sand off sharp corners.
 
-If you're up for contributing and don't have a preference for writing code or
-not, this is the area that would most benefit from your contribution. Issues are
-tagged with
-[documentation](https://github.com/prql/prql/labels/documentation).
+Both bug reports of unfriendliness, and code contributions to improve them are welcome; there's a
+[friendliness label.](https://github.com/prql/prql/issues?q=is%3Aissue+label%3Afriendlienss+is%3Aopen)
 
-## Friendliness
+## Standard library
 
-Currently the language implementation is not sufficiently friendly, despite
-significant recent improvements. We'd like to make error messages better, sand
-off sharp corners, etc.
+Currenty, standard library is [quite limited](https://github.com/prql/prql/blob/main/prql-compiler/src/sql/stdlib.prql).
+It contains only basic arithmetic functions (AVERAGE, SUM) and lacks functions for string manipulation,
+date handling and many math functions.
+Problem here is that PRQL is limited with functionality of target DBMS and its capabilities and the amount
+of different dialects.
 
-Both bug reports of unfriendliness, and code contributions to improve them are
-welcome; there's a
-[friendliness](https://github.com/prql/prql/issues?q=is%3Aissue+label%3Afriendlienss+is%3Aopen)
-label.
+Before that, we need to setup a testing framework that would run queries against actual databases,
+so we know that the dialect implementation is on-point.
```

Comment:
```suggestion
```

Covered above (and I think we should try and not put up blockers, like "before that" — if we actually hit problems then we can slow down, but let's not do any prior restraint.